Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -345,19 +345,22 @@ def request(self, path=None, params=None, headers=None, authenticated=None, pagi
             raise self.Exception('Request requires authentication, none provided')
 
         params = {key: value for key, value in params.items()} if params else dict()
-        params['per_page'] = params.get('per_page', 100)
-        params['page'] = params.get('page', 1)
+        if paginate:
+            params['per_page'] = params.get('per_page', 100)
+            params['page'] = params.get('page', 1)
+
+        json = {key: value for key, value in json.items()} if json else dict()
```

Comment:
Shouldn't this just be "None" if no json is defined? Otherwise we'll send an empty dictionary to on every `GET` request?

Reply:
Done.